settings.gradle.kts

```diff
@@ -0,0 +1,4 @@
+rootProject.name = "WalletSdk"
+
+/** SDK */
+include("core")
```

Comment:
I don't love the `core` name. Is there any issue with just naming it `wallet-sdk`? I don't think we should split wallet SDK functionality between subprojects, so that should be fine right?

Reply:
I will rename it to `wallet-sdk`. 
Also, for now the artifact for maven (to be used by external folks in the future) has name of
`org.stellar.wallet-sdk:core:0.1.0` 
I think I should rename it to
`org.stellar:wallet-sdk:0.1.0` as well?